### PR TITLE
Add all yabai supported locations

### PR DESCRIPTION
**Pull request by Liu233w**
The locations are listed in its wiki: https://github.com/koekeishiya/yabai/wiki/Configuration#configuration-file

### All submissions

* [x] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
* [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change

### Adding/updating Application X Support

* [x] This PR is only for one application
* [x] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
* [x] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
* [x] Syncing does not break the application
* [x] Syncing does not compete with any syncing functionality internal to the application
* [x] The configuration syncs the minimal set of data
* [x] No file specific to the local workstation is synced
* [x] No sensitive data is synced

### Improving the Mackup codebase

* [ ] My submission passes the [tests](https://github.com/lra/mackup/tree/master/tests)
* [ ] I have linted the code locally prior to submission
* [ ] I have written new tests as applicable
* [ ] I have added an explanation of what the changes do

### CODE REVIEW

File: mackup/applications/yabai.cfg [hunk 1]
```diff
@@ -3,3 +3,7 @@ name = yabai
 
 [configuration_files]
 .yabairc
+.config/yabai/yabairc
```
Comment (lra) [inline mackup/applications/yabai.cfg:6]: As you added `xdg_configuration_files`, you can remove this, it would duplicate `.config/yabai/yabairc`.

**Review by lra — CHANGES_REQUESTED** [review]

File: mackup/applications/yabai.cfg (hunk 1 above)
Comment (Liu233w) [reply mackup/applications/yabai.cfg:6]: According to yabai's document, the path `$HOME/.config/yabai/yabairc` is also supported, and I use that as my config path. Although my `XDG_CONFIG_HOME` is empty.

Comment (lra) [reply mackup/applications/yabai.cfg:6]: `.config/yabai/yabairc` is the default XDG config when `XDG_CONFIG_HOME` is unset.

Comment (Liu233w) [reply mackup/applications/yabai.cfg]: Thanks for the review. I just updated the file.

**Review by lra — APPROVED** [review]
Thanks!

**Pull request merged**